Whisky/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -1,6 +1,10 @@
 {
   "colors" : [
     {
+      "color" : {
```

Comment:
Remove this addition

Reply:
Will do.

Reply:
I really liked that accent color matching whisky though ;(